# DIETA NUTRICIONAL 

### VALORES NUSTRICONES

###### Daniel Eduardo Acuña Ballen   cod:161003700
###### Andres Camilo Gamboa Ferrer cod:161003711

$los$ $siguientes$ $valores$ $de$ $nutricion$, $son$ $los$ $recomendables$ $en$ $el$ $consumo$ $diario$ $de$ $un$ $persona$ $y$ $se$ $muestra$ $en$ $la$ $siguiente$ $tabla:$

|                |   Calorias| 2000     |
| :------------: |:---------:|:-------  |
| Grasa total    | menos de  | 65g    	|
| Grasa saturada | menos de  | 20g    	|
| Colesterol     | menos de  | 300mg  	|
| sodio          | menos de  | 2400mg 	|
| Carbohidratos  | menos de  | 300g   	|
| Fibra dietaria | menos de  | 25g    	|
| Proteina       |           | 200g     |
| Vitamina A     |           | 900mcg   |
| Vitamina C     |           | 90mg     |
| Calcio         |           | 2500mg   |
| Hierro         |           | 18mg     |
$Tabla$ $N°1$ $Valor$ $nutricional$ $diario$

$Se$ $tiene$ $los$ $diferentes$ $productos$ $mostrados$ $a$ $continuacion$ $para$ $planear$ $una$ $dieta$ $balanceada$, $basandose$ $en$ $la$ $tabla$ $anterior$:


#### GALLETAS 

|                | calorias  |  120     |
| :------------: |:---------:|:-------  |
| Grasa total    | 9%        | 6g    	|
| Grasa saturada | 15%       | 3g    	|
| Colesterol     | 0%        | 0mg  	|
| sodio          | 0%        | 10mg 	|
| Carbohidratos  | 5%        | 16g   	|
| Fibra dietaria | 0         | 0g    	|
| Proteina       | 1%        | 2g       |
| Vitamina A     | 0%        | 0g       |
| Vitamina C     | 0%        | 0g     	|
| Calcio         | 0%        | 0g      	|
| Hierro         | 4%        | 0.72mg   |
$Tabla$ $N°2$ $nutrientes$ $por$ $24g$ $galletas$


$Como$ $en$ $un$ $paquete$ $hay$ $80g$ $de$ $galletas$ $entonces$ $la$ $cantidad$ $de$ $calorias$ $que$ $hay$ $en$ $un$ $paquete$ $sera$ $la$ $siguiente$, $asi$ $como$ $sus$ $otros$ $factores$ $nutricionales$:
    

In [5]:
galletas_1=3;
Gramos_total=24;
caloT=(galletas_1*80)/Gramos_total
n1=caloT/3
grasa=6*n1
grasasat=3*n1
sodio=0*n1;
carbohid=16*n1
fribde=0*n1
proteina=2*n1
Va=n1*0
Vc=n1*0
Ca=n1*0
H=n1*0.72

print("Calorias:")
print(n1*120)
print("\n")
print("Grasa total:")
print(grasa,"g")
print("\n")
print("Grasa Saturada:")
print(grasasat,"g")
print("\n")
print("Sodio:")
print(sodio,"g")
print("\n")
print("Carbohidratos:")
print(carbohid,"g")
print("\n")
print("Fibra dietaria:")
print(fribde,"g")
print("\n")
print("Proteina:")
print(proteina,"g")
print("\n")
print("Vitamina A:")
print(Va,"mcg")
print("\n")
print("Vitamina C:")
print(Vc,"mg")
print("\n")
print("Calcio:")
print(Ca,"mg")
print("\n")
print("Hierro:")
print(H,"mg")
print("\n")







Calorias:
400.0


Grasa total:
20.0 g


Grasa Saturada:
10.0 g


Sodio:
0.0 g


Carbohidratos:
53.333333333333336 g


Fibra dietaria:
0.0 g


Proteina:
6.666666666666667 g


Vitamina A:
0.0 mcg


Vitamina C:
0.0 mg


Calcio:
0.0 mg


Hierro:
2.4 mg




#### ARANDANOS

|                | calorias  |  110     |
| :------------: |:---------:|:-------  |
| Grasa total    | 2%        | 1g    	|
| Grasa saturada | 0%        | 0g    	|
| Colesterol     | 0%        | 0mg  	|
| sodio          | 0%        | 0mg 	    |
| Carbohidratos  | 8%        | 23g   	|
| Fibra dietaria | 8%        | 2g    	|
| Proteina       | 1%        | 2g       |
| Vitamina A     | 2%        | 18mcg    |
| Vitamina C     | 15%       | 13.5mg   |
| Calcio         | 0%        | 0g       |
| Hierro         | 0%        | 0g       |
$Tabla$ $N°3$ $nutrientes$ $por$ $30g$ $arandanos$

$Como$ $en$ $un$ $paquete$ $hay$ $90g$ $de$ $arandanos$ $entonces$ $la$ $cantidad$ $de$ $calorias$ $que$ $hay$ $en$ $un$ $paquete$ $sera$ $la$ $siguiente$, $asi$ $como$ $sus$ $otros$ $factores$ $nutricionales$:
    

In [6]:
arandanos_1=3;
Gramos_total=30;
caloT=(arandanos_1*90)/Gramos_total
n1=caloT/3
grasa=1*n1
grasasat=0*n1
sodio=0*n1;
carbohid=23*n1
fribde=2*n1
proteina=2*n1
Va=n1*18
Vc=n1*13.5
Ca=n1*0
H=n1*0

print("Calorias:")
print(n1*110)
print("\n")
print("Grasa total:")
print(grasa,"g")
print("\n")
print("Grasa Saturada:")
print(grasasat,"g")
print("\n")
print("Sodio:")
print(sodio,"g")
print("\n")
print("Carbohidratos:")
print(carbohid,"g")
print("\n")
print("Fibra dietaria:")
print(fribde,"g")
print("\n")
print("Proteina:")
print(proteina,"g")
print("\n")
print("Vitamina A:")
print(Va,"mcg")
print("\n")
print("Vitamina C:")
print(Vc,"mg")
print("\n")
print("Calcio:")
print(Ca,"mg")
print("\n")
print("Hierro:")
print(H,"mg")
print("\n")

Calorias:
360.0


Grasa total:
3.0 g


Grasa Saturada:
0.0 g


Sodio:
0.0 g


Carbohidratos:
69.0 g


Fibra dietaria:
6.0 g


Proteina:
6.0 g


Vitamina A:
54.0 mcg


Vitamina C:
40.5 mg


Calcio:
0.0 mg


Hierro:
0.0 mg




#### PAN DE ARROZ

|                | calorias  |  140K    |
| :------------: |:---------:|:-------  |
| Grasa total    | 8%        | 5g    	|
| Grasa saturada | 15%       | 3g    	|
| Colesterol     | 2%        | 5mg  	|
| sodio          | 13%       | 320mg 	|
| Carbohidratos  | 6%        | 17g   	|
| Fibra dietaria | 8%        | 2g    	|
| Proteina       | 3%        | 6g       |
| Vitamina A     | 0%        | 0g       |
| Vitamina C     | 0%        | 0g       |
| Calcio         | 10%       | 375mg    |
| Hierro         | 0%        | 0g       |
$Tabla$ $N°4$ $nutrientes$ $por$ $30g$ $rosquillas$

$Como$ $en$ $un$ $paquete$ $hay$ $78g$ $de$ $galletas$ $entonces$ $la$ $cantidad$ $de$ $calorias$ $que$ $hay$ $en$ $un$ $paquete$ $sera$ $la$ $siguiente$, $asi$ $como$ $sus$ $otros$ $factores$ $nutricionales$:

In [4]:
rosquillas_1=9
Gramos_total=30
caloT=(rosquillas_1*78)/Gramos_total
n1=caloT/9
grasa=5*n1
grasasat=3*n1
coles=5*n1
sodio=320*n1
carbohid=17*n1
fribde=2*n1
proteina=6*n1
Va=n1*0
Vc=n1*0
Ca=n1*375
H=n1*0

print("Calorias:")
print(n1*140,"K")
print("\n")
print("Grasa total:")
print(grasa,"g")
print("\n")
print("Grasa Saturada:")
print(grasasat,"g")
print("\n")
print("Colesterol:")
print(coles,"mg")
print("\n")
print("Sodio:")
print(sodio,"mg")
print("\n")
print("Carbohidratos:")
print(carbohid,"g")
print("\n")
print("Fibra dietaria:")
print(fribde,"g")
print("\n")
print("Proteina:")
print(proteina,"g")
print("\n")
print("Vitamina A:")
print(Va,"mcg")
print("\n")
print("Vitamina C:")
print(Vc,"mg")
print("\n")
print("Calcio:")
print(Ca,"mg")
print("\n")
print("Hierro:")
print(H,"mg")
print("\n")

Calorias:
363.99999999999994 K


Grasa total:
12.999999999999998 g


Grasa Saturada:
7.799999999999999 g


Colesterol:
12.999999999999998 mg


Sodio:
831.9999999999999 mg


Carbohidratos:
44.199999999999996 g


Fibra dietaria:
5.199999999999999 g


Proteina:
15.599999999999998 g


Vitamina A:
0.0 mcg


Vitamina C:
0.0 mg


Calcio:
974.9999999999999 mg


Hierro:
0.0 mg




#### COMPOTA

|                | calorias  |  80      |
| :------------: |:---------:|:-------  |
| Grasa total    | 0%        | 0g    	|
| Grasa saturada | 0%        | 0g    	|
| Colesterol     | 0%        | 5mg  	|
| sodio          | 1%        | 20mg 	|
| Carbohidratos  | 7%        | 20g   	|
| Fibra dietaria | 0%        | 0g    	|
| Proteina       | 0%        | 0g       |
| Vitamina A     | 0%        | 0g       |
| Vitamina C     | 50%       | 45mg     |
| Calcio         | 9%        | 225mg    |
| Hierro         | 0%        | 0g       |
$Tabla$ $N°5$ $nutrientes$ $frasco$ $de$ $compota$ $113g$ 

#### PALOTRIQUES

|                | calorias  |  80      |
| :------------: |:---------:|:-------  |
| Grasa total    | 2%        | 1g    	|
| Grasa saturada | 0%        | 0g    	|
| Colesterol     | 0%        | 5mg  	|
| sodio          | 4%        | 130mg 	|
| Carbohidratos  | 3%        | 10g   	|
| Fibra dietaria | 0%        | 0g    	|
| Proteina       | 1%        | 2g       |
| Vitamina A     | 0%        | 0g       |
| Vitamina C     | 0%        | 0g       |
| Calcio         | 0%        | 0g       |
| Hierro         | 4%        | 0.72mg   |
$Tabla$ $N°6$ $nutrientes$ $frasco$ $de$ $compota$ $14g$ 

$Como$ $en$ $un$ $paquete$ $hay$ $116.2g$ $de$ $palitriques$ $entonces$ $la$ $cantidad$ $de$ $calorias$ $que$ $hay$ $en$ $un$ $paquete$ $sera$ $la$ $siguiente$, $asi$ $como$ $sus$ $otros$ $factores$ $nutricionales$:


In [7]:
palitos=8.3
Gramos_total=14
caloT=(palitos*116.2)/Gramos_total
n1=caloT/8.3
grasa=1*n1
grasasat=0*n1
coles=5*n1
sodio=130*n1
carbohid=10*n1
fribde=0*n1
proteina=2*n1
Va=n1*0
Vc=n1*0
Ca=n1*0
H=n1*0.72

print("Calorias:")
print(n1*80)
print("\n")
print("Grasa total:")
print(grasa,"g")
print("\n")
print("Grasa Saturada:")
print(grasasat,"g")
print("\n")
print("Colesterol:")
print(coles,"mg")
print("\n")
print("Sodio:")
print(sodio,"mg")
print("\n")
print("Carbohidratos:")
print(carbohid,"g")
print("\n")
print("Fibra dietaria:")
print(fribde,"g")
print("\n")
print("Proteina:")
print(proteina,"g")
print("\n")
print("Vitamina A:")
print(Va,"mcg")
print("\n")
print("Vitamina C:")
print(Vc,"mg")
print("\n")
print("Calcio:")
print(Ca,"mg")
print("\n")
print("Hierro:")
print(H,"mg")
print("\n")

Calorias:
1162.0 K


Grasa total:
8.3 g


Grasa Saturada:
0.0 g


Colesterol:
41.5 mg


Sodio:
1079.0 mg


Carbohidratos:
83.0 g


Fibra dietaria:
0.0 g


Proteina:
16.6 g


Vitamina A:
0.0 mcg


Vitamina C:
0.0 mg


Calcio:
0.0 mg


Hierro:
5.976 mg




### PRECIO DE LOS PRODUCTO POR PAQUETES

| PRODUCTO       | CANTIDAD  | PRECIO($)   |
| :------------: |:---------:|:-------  |
| Arandanos      | 100g      |  3100   |
| Pan de arroz   | 80g       |  1950   |
| Wafer minitaco | 80g       |  1050   |
| Minipalitrooque| 100g      |  1800   |
| Compota        |           |  1250   |

$Tabla$ $N°6$ $precios$ $de$ $los$ $productos$  

##### PROBLEMA A RESOLVER

$Por$ $medio$ $de$ $la$ $tabla$ $se$ $debe$ $plantear$ $una$ $dieta$ $nutricional$ $balanceada$ $diaria$ $que$ $cumpla$ $con$ $restriciones$ $de$ $nutrientes$ $maximo$ $que$ $se$ $basan$ $en$ $ 2000 $ $calorias$ $diarias$, $donde$ $se$ $estudian$ $los$ $nutrientes$ $por$ $paquete$

##### VARIABLES



$$
X0=Pan de arroz
$$
$$
X1=Arandanos
$$
$$
X2=Wafer minitaco
$$
$$
X3=Minipalitroowe
$$
$$
X4=Compota
$$

$Donde$ $se$ $debe$ $tener$ $la$ $siguientes$ $condiciones$,$las$ $condiciones$ $estan$ $basadas$ $en$ $gramos$

##### GRASA TOTAL
$$
12.9X0+3X1+20X2+8.3X3\geq  65 \\
$$
##### GRASA SATURADA
$$
7.79X0+10X2\geq 20\\
$$
##### COLESTEROL
$$
0.0129X0+0.0415X3+0.005X4\geq 0.3\\
$$
##### SODIO
$$
0.8319X0+1.079X3+0.225X4\geq 2.4 \\
$$
##### CARBOHIDRATOS
$$
44.1X0+69X1+53.3X2+83X3\geq 300\\
$$
##### FIBRA DIETARIA
$$
5.19X0+6X1\geq 25 \\
$$
##### VITAMINA A
$$
0.000054X0\geq 0.0009\\
$$
##### VITAMINA C
$$
0.0405X1+0.045X4\geq 0.09\\
$$
##### HIERRO
$$
0.005976X3\geq 0.018\\
$$
##### CALCIO
$$
0.97499X0+0.225X4\geq 2.5 \\
$$

##### FUNCION OBJETIVO

$Funcion$ $objetivo$
$$
f=1950X0+3100X1+1050X2+1800X3+1250X4
$$

...[1].